permissions/new/ProvisioningInfo.json

```
@@ -153,7 +153,7 @@
         "isEnabled": true,
         "resourceAppId": "00000003-0000-0000-c000-000000000000"
       }
-    ],
+    ],      
```

Should not be introducing trailing whitespace

```suggestion
    ],
```